From d16094774b8e709d90ad33530fbfadecec6ec825 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 10 Sep 2019 15:49:39 +0200
Subject: [PATCH 035/152] build: Remove st-private.h from "public" headers

It is currently listed both as public and private header, which is
clearly bonkers. As the name implies, the latter is correct ...

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/709
---
 src/st/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/st/meson.build b/src/st/meson.build
index 764ff0db4..4cbebace7 100644
--- a/src/st/meson.build
+++ b/src/st/meson.build
@@ -14,7 +14,6 @@ st_headers = [
   'st-icon-colors.h',
   'st-image-content.h',
   'st-label.h',
-  'st-private.h',
   'st-scrollable.h',
   'st-scroll-bar.h',
   'st-scroll-view.h',
-- 
2.23.0

